source/common/router/route_config_update_receiver_impl.cc

```diff
@@ -79,5 +97,10 @@ void RouteConfigUpdateReceiverImpl::rebuildRouteConfig(
   }
 }
 
+bool RouteConfigUpdateReceiverImpl::isAliasNotResolvedReply(
```

Comment:
This name took me a few seconds to understand. It sounds like a reply to an inquiry of whether the alias is not resolved. How about `aliasResolutionFailed`?

Reply:
done